gen/go/collector/metrics/v1/metrics_service.pb.go

```diff
@@ -132,11 +132,11 @@ var fileDescriptor_75fb6015e6e64798 = []byte{
 
 // Reference imports to suppress errors if they are not otherwise used.
 var _ context.Context
-var _ grpc.ClientConn
+var _ grpc.ClientConnInterface
 
 // This is a compile-time assertion to ensure that this generated file
 // is compatible with the grpc package it is being compiled against.
-const _ = grpc.SupportPackageIsVersion4
+const _ = grpc.SupportPackageIsVersion6
```

Comment:
Not sure why the gRPC version went from 4 to 6. Possibly a requirement of gRPC gateway?

Reply:
Fixed, generated using correct version of gen tools.